enginetest/queries/query_plans.go

```diff
@@ -7041,6 +7041,7 @@ inner join pq on true
 			"",
 	},
 	{
+		// TODO this is an invalid query
```

Comment:
Add a skipped test

Reply:
as in it errors in mysql, will need to switch to error test for new name res